ThirdPartyNotices.txt

```diff
@@ -0,0 +1,556 @@
+
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+App Center Unity SDK incorporates third party material from the projects listed below. The original copyright notice and the license under which Microsoft received such third party material are set forth below. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.
```

Comment:
@annakocheshkova Unity SDK? Should be cordova I believe

Reply:
:+1: